src/csvcubed/utils/sparql_handler/sparql_queries/select_unit_col_observed_col_title_and_about_url.sparql

```diff
@@ -0,0 +1,27 @@
+PREFIX qb: <http://purl.org/linked-data/cube#>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX csvw: <http://www.w3.org/ns/csvw#>
+
+SELECT ?csvUrl ?observationValueColumnTitle ?observationValueColumnAboutUrl
+WHERE {
+    ?table csvw:tableSchema ?tableSchema;
+           csvw:url ?csvUrl.
+        
+    ?tableSchema csvw:column/rdf:rest*/rdf:first ?observedValueColumn.
+    FILTER NOT EXISTS {
+        ?observedValueColumn csvw:virtual true.
+    }
+
+    OPTIONAL{
+        ?observedValueColumn csvw:aboutUrl ?observationValueColumnAboutUrl.
+    }
+    ?observedValueColumn csvw:propertyUrl ?observedValueColPropertyUrl;
+                         csvw:title ?observationValueColumnTitle.
+
+    FILTER EXISTS {
+            ?dsd_uri a qb:DataStructureDefinition;
```

Comment:
We may need to think about bringing ?dsd_uri back out of the query as well and make a double dictionary. We should discuss this a bit later.